src/utils/hooks/useDocumentTitle.jsx

```diff
@@ -0,0 +1,7 @@
+import { useEffect } from 'react'
+
+export default function useDocumentTitle(title){
+    useEffect(()=>{
+        document.title = title
+    },[])

```

Comment:
I think title param should be added to dependency array.

Then if you pass title as a constant it will update only once on page load , if you pass a state variable it will update on that state change

Reply:
Thanks for mentioning this. Added title to dependency array.